### PR TITLE
Enhanced App Mesh control/data plane Grafana dashboards

### PR DESCRIPTION
**Pull request by fawadkhaliq**
**Description of changes**
Enhanced the App Mesh EKS integration control plane and data plane Grafana dashboards.

*Data plane dashboard*
- Added connection timeout per namespace
- Added request timeout per namespace
- Fixed the memory and CPU usage dashboard. Metrics names have been changed starting k8s 1.1.6 (see the details [here](https://github.com/kubernetes/kubernetes/blob/master/CHANGELOG/CHANGELOG-1.16.md#removed-metrics))
- Added Envoy 2xx and 4xx/5xx HTTP metrics

*Control plane dashboard*
- Added goroutines metrics
- Added AWS API calls 2xx, 4xx/5xx metrics
- Added work queue unfinished work metrics
- Fixed the memory and CPU usage dashboard. Metrics names have been changed starting k8s 1.1.6 (see the details [here](https://github.com/kubernetes/kubernetes/blob/master/CHANGELOG/CHANGELOG-1.16.md#removed-metrics))

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: stable/appmesh-grafana/dashboards/control-plane.json [hunk 1]
```diff
@@ -1,415 +1,681 @@
...
+  "graphTooltip": 0,
+  "links": [],
+  "panels": [
+    {
+      "aliasColors": {},
+      "bars": false,
+      "dashLength": 10,
+      "dashes": false,
+      "datasource": null,
+      "description": "Total number of SDK API calls from the customer's code to AWS services",
```
Comment (achevuru) [inline stable/appmesh-grafana/dashboards/control-plane.json:27]: By customer code do you mean customer's application code? These are the AWS API call stats from the controller right?

File: stable/appmesh-grafana/dashboards/control-plane.json [hunk 2]
```diff
@@ -1,415 +1,681 @@
...
-    "graphTooltip": 0,
-    "links": [],
-    "panels": [
+    {
+      "aliasColors": {},
+      "bars": false,
+      "dashLength": 10,
+      "dashes": false,
+      "datasource": null,
+      "description": "",
```
Comment (achevuru) [inline stable/appmesh-grafana/dashboards/control-plane.json:115]: Good to have a short desc. Maybe "Successful API calls" or..

File: stable/appmesh-grafana/dashboards/control-plane.json (hunk 1 above)
Comment (fawadkhaliq) [reply stable/appmesh-grafana/dashboards/control-plane.json:27]: Ah this is a misleading description. Let me fix it

File: stable/appmesh-grafana/dashboards/control-plane.json [hunk 3]
```diff
@@ -1,415 +1,681 @@
...
+        "align": false,
+        "alignLevel": null
+      }
+    },
+    {
+      "aliasColors": {},
+      "bars": false,
+      "dashLength": 10,
+      "dashes": false,
+      "datasource": null,
```
Comment (achevuru) [inline stable/appmesh-grafana/dashboards/control-plane.json:202]: Same here...description field would be good.

File: stable/appmesh-grafana/dashboards/control-plane.json [hunk 4]
```diff
@@ -1,415 +1,681 @@
...
+  "gnetId": null,
+  "graphTooltip": 0,
+  "links": [],
+  "panels": [
+    {
+      "aliasColors": {},
+      "bars": false,
+      "dashLength": 10,
+      "dashes": false,
+      "datasource": null,
```
Comment (achevuru) [inline stable/appmesh-grafana/dashboards/control-plane.json:26]: Isn't the datasource prometheus for this? and for all the metrics here in general?

Comment (fawadkhaliq) [reply stable/appmesh-grafana/dashboards/control-plane.json:26]: datasource: null means use the datasource marked as default. We don't want to hardcode to a specific source name as the name can be changed or customer may have their own Prometheus configured

Comment (achevuru) [reply stable/appmesh-grafana/dashboards/control-plane.json:26]: Yes, but my concern was that some point to prometheus and some point to null

Comment (fawadkhaliq) [reply stable/appmesh-grafana/dashboards/control-plane.json:26]: Fixed it. There were leftovers

**Review by achevuru — APPROVED** [review]

**Pull request merged**